src/Entities/Entity.cpp

```diff
@@ -1674,11 +1674,11 @@ void cEntity::SetSwimState(cChunk & a_Chunk)
 			{
 				BLOCKTYPE Block;
 				if (!a_Chunk.UnboundedRelGetBlockType(x, y, z, Block))
-				{
+				{ /*
 					LOGD("SetSwimState failure: RelX = %d, RelY = %d, RelZ = %d, Pos = %.02f, %.02f}",
 						x, y, z, GetPosX(), GetPosZ()
 					);
-					continue;
+					continue; */
```

Comment:
Why disable the continue?

Reply:
whoops - didn't meen to commit. The setswimstate stuff is quite annoying if you try to use the console for something